tool_collections/samtools/sam_to_bam/sam_to_bam.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="sam_to_bam" name="SAM-to-BAM" version="1.1.6">
-  <description>converts SAM format to BAM format</description>
+<tool id="sam_to_bam" name="SAM-to-BAM" version="1.1">
```

Comment:
Here the version number is decreased (on main Tool Shed latest version is 1.1.4).